README.md

```
@@ -124,8 +124,8 @@ We are a crew of problem-solvers, designers, and builders, and we're super excit
         <table align="center">
           <tr>
             <td align="center" width="150">
-                <a href="https://umd.hack4impact.org/">
-                <img src="frontend/src/assets/winrock-profile-pictures/pending-image.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
+                <a href="https://www.linkedin.com/in/nadia-meyerovich-4a30a9215/">
+                <img src="frontend/src/assets/winrock-profile-pictures/nadia-meyerovich-headshot.png" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)